qutebrowser/misc/crashdialog.py

```diff
@@ -624,3 +642,8 @@ def dump_exception_info(exc, pages, cmdhist, objects):
     print('\n'.join(cmdhist), file=sys.stderr)
     print("\n---- Objects ----", file=sys.stderr)
     print(objects, file=sys.stderr)
+    print("\n---- Environment ----", file=sys.stderr)
+    try:
+        print(_get_environment_vars(), file=sys.stderr)
+    except Exception:
+        traceback.print_exc()
```

Comment:
Oh, you're right - I didn't view enough of the diff, sorry. Let's leave it in then.